src/importexport/finale/internal/text/fontdata/MaestroTimes.json

```
@@ -0,0 +1,62 @@
+	"metNote8thUp": {
+		"codepoint": "U+ECA7",
+		"legacyCodepoint": "137",
+		"description": "Eighth note (quaver) stem up"
+	},
+	"textTuplet3LongStem": {
+		"codepoint": "U+E202",
+		"legacyCodepoint": "205",
+		"index": 0,
+		"description": "Tuplet number 3 for long stem"
+	},
+	"textBlackNoteFrac8thLongStem": {
```

Biome (2.5.3)

[error] The key textBlackNoteFrac8thLongStem was already declared.

(lint/suspicious/noDuplicateObjectKeys)